testr/testr.go

```diff
@@ -116,36 +112,11 @@ func logError(t TestingT, formatError func(error, string, []interface{}) (string
 }
 
 type testlogger struct {
```

Comment:
As per the below discussion, we should comment here that this exists to wrap and modify the method-set of testloggerInterface, in particular it changes the GetUnderlying() method